src/content/plots/PlotDownload.service.tsx

```diff
@@ -0,0 +1,56 @@
+import downloadjs from 'downloadjs'
+import html2canvas from 'html2canvas'
+import * as d3 from 'd3'
+
+const PlotDownloadService = {
+    downloadAsPNG: async function () {
```

Comment:
```suggestion
    downloadAsPNG: async () => {
```
Since arrow functions are used elsewhere in this file, it would be better to be consistent in their usage